src/com/twitter/intellij/pants/execution/PantsMakeBeforeRun.java

```diff
@@ -199,6 +199,7 @@ public boolean executeTask(
     // If project has not changed since last Compile, return immediately.
     if (!FileChangeTracker.shouldRecompileThenReset(currentProject, targetAddressesToCompile)) {
       PantsExternalMetricsListenerManager.getInstance().logIsPantsNoopCompile(true);
+      showPantsMakeTaskMessage("Already up to date.\n", ConsoleViewContentType.SYSTEM_OUTPUT, currentProject);
```

Comment:
will change it to a notification since printing out in console may not be obvious either.